unumpy/tests/test_numpy.py

```diff
@@ -326,6 +346,16 @@ def test_functions_coerce_with_dtype(backend, method, args, kwargs):
         (np.dsplit, ([[[1, 2], [3, 4]], [[5, 6], [7, 8]]], 2), {}),
         (np.hsplit, ([[1, 2], [3, 4]], 2), {}),
         (np.vsplit, ([[1, 2], [3, 4]], 2), {}),
+        (np.ix_, ([0, 1], [2, 4]), {}),
+        (np.unravel_index, ([22, 41, 37], (7, 6)), {}),
```

Comment:
This needs an `xfail`. It's broken on cupy's side.

Reply:
What `reason` message do you think it's appropriate?

Reply:
"cupy.unravel_index is broken in version 6.0"

Reply:
`ix_` is working or broken as well?

Reply:
`ix_` works.

Reply:
Also, what error is being raised by `cupy.unravel_index` (e.g., `ValueError`, `TypeError`)?

Reply:
`TypeError`.